.prettierignore

```diff
@@ -0,0 +1 @@
+**/*.mdx
```

Comment:
I thought that Prettier understood MDX but it's basically destroying all of these components when enabled:

![Screenshot 2025-03-12 at 2 34 35 PM](https://github.com/user-attachments/assets/2f51cf0d-8b14-4c33-b413-a7c2e94e9b3d)